From b482374db0cfed629ad2e092705dd26b4cd8f8fe Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Thu, 19 Jun 2014 23:36:50 +0100
Subject: [PATCH 1/5] binarycreator: add qico and qtaccessiblewidgets as
 QTPLUGINs

Commit 807de076f4fee6c relies on the autoplugs mechanism (added in
qtbase commit 4255ba40ab073af) to add the needed statically linked
plugins; however because binarycreator is a console app it doesn't
depend on gui (QT -= gui) this mechanism fails to add the plugins.
---
 tools/binarycreator/binarycreator.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/binarycreator/binarycreator.pro b/tools/binarycreator/binarycreator.pro
index 674fbdb..7492647 100644
--- a/tools/binarycreator/binarycreator.pro
+++ b/tools/binarycreator/binarycreator.pro
@@ -5,11 +5,11 @@ INCLUDEPATH += . .. rcc ../common
 include(../../installerfw.pri)
 
 QT -= gui
+QTPLUGIN += qico qtaccessiblewidgets
 QT += qml xml
 
 CONFIG += console
 DESTDIR = $$IFW_APP_PATH
-
 SOURCES = binarycreator.cpp \
           rcc/rcc.cpp \
           rcc/rccmain.cpp \
-- 
2.0.0

